test/common/grpc/codec_test.cc

```diff
@@ -106,6 +107,41 @@ TEST(GrpcCodecTest, decodeInvalidFrame) {
   EXPECT_EQ(size, buffer.length());
 }
 
+// A frame of nothing followed by a frame of 0's followed then invalid
+TEST(GrpcCodecTest, decodeNullByteFrameWithInvalidFrameAfterward) {
+  const std::string data("\000\000\000\000\0000000", 9);
+  Buffer::OwnedImpl buffer(data.data(), data.size());
+  std::vector<Frame> frames;
+  Decoder decoder;
+  ENVOY_LOG_MISC(trace, "Buffer: {}", buffer.toString());
+  EXPECT_FALSE(decoder.decode(buffer, frames));
+  // When the decoder doesn't successfully decode, it puts valid frames up until
```

Comment:
im not sure i understand this comment in this context

Reply:
Switched from "valid frame" to "decoded frame", as it's questionable whether the codec should really parse these null bytes as a frame.